### PR TITLE
JP-370 Support for IDMS fixed links

### PR DESCRIPTION
**Pull request by pies**

### CODE REVIEW

File: src/database/MySQLTable.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export class MySQLTable {
     this.buffer[row.action].push(row);
 
     if (this.buffer[row.action].length >= this.flushLimit) {
-      return this.flush(row.action);
+      return await this.flush(row.action);
```
Comment (pies) [inline src/database/MySQLTable.ts]: Was this omission intentional?

File: src/index.ts [hunk 2]
```diff
@@ -3,7 +3,12 @@ import {Container} from "./cli/Container";
 
 const container = new Container();
 
+let command = process.argv[2];
```
Comment (pies) [inline src/index.ts]: To allow using as `npm run start gtfs`.

File: src/database/MySQLTable.ts (hunk 1 above)
Comment (linusnorton) [reply src/database/MySQLTable.ts]: Adding await there doesn't do anything unless it's wrapped in a try catch statement. Await just yields a promise, so there you are yielding, then continuing once the promise is resolved and returning the value. Returning the promise of something is the same

**Comment by linusnorton** [comment]
Hi, thanks for the PR but I don't think I want this functionality at the moment. I suggest you fork and maintain the @assertis/dtd2mysql version.

**Comment by pies** [comment]
@linusnorton That was my feeling as well, but I wanted to make sure :)

File: src/database/MySQLTable.ts (hunk 1 above)
Comment (pies) [reply src/database/MySQLTable.ts]: Without it the code that called it returned immediately. With it it waited for the flush to be complete. I could not get it to work correctly without the await here.

**Pull request closed without merging**